marketplace/gatsby-preview/src/AppConfig.js

```diff
@@ -14,60 +15,70 @@ import {
 import GatsbyIcon from './GatsbyIcon';
 import styles from './styles';
 
+function editorInterfacesToEnabledContentTypes(eis, appId) {
+  const findAppWidget = item => item.widgetNamespace === 'app' && item.widgetId === appId;
+
+  return eis
+    .filter(ei => !!get(ei, ['sidebar'], []).find(findAppWidget))
+    .map(ei => get(ei, ['sys', 'contentType', 'sys', 'id']))
+    .filter(ctId => typeof ctId === 'string' && ctId.length > 0);
```

Comment:
I'm seeing some similar logic as in the other apps. Would it be worth abstracting this into an `sdk.app` method for convenience to the end-user?

Reply:
@davidfateh Yes, absolutely, I'm fully aware of the repetition. This is a part of a bigger problem: we still use `targetState` in all the apps (which is not a part of the final interface).

I don't want to introduce these "convenience" methods to the SDK since they are usecase-specific and I don't want to grow the SDK any further.

BUT the idea would be to introduce a new package (think `shared-assignment-helpers`) that would allow:

1. read `EditorInterface`s (any any other assignment entities we'll add in the future) and convert them to a pleasant data structure (for UI rendering)
2. save changes introduced in the data structure (1) back to assignment entities (so the things that is done w/ `targetState` now)
3. potentially it could export a set of components for rendering assignment UIs, but for Jan/Feb it would be a strech

That being said there's some work to do and I want to tackle it separately.